zerver/models.py

```diff
@@ -2221,6 +2221,39 @@ class Stream(models.Model):
     # Foreign key to the Recipient object for STREAM type messages to this stream.
     recipient = models.ForeignKey(Recipient, null=True, on_delete=models.SET_NULL)
 
+    # Various permission policy configurations
+    PERMISSION_POLICIES: Dict[str, Dict[str, Any]] = {
+        "web_public": {
+            "invite_only": False,
+            "history_public_to_subscribers": True,
+            "is_web_public": True,
+            "policy_name": gettext_lazy("Web public"),
+        },
+        "public": {
+            "invite_only": False,
+            "history_public_to_subscribers": True,
+            "is_web_public": False,
+            "policy_name": gettext_lazy("Public"),
+        },
+        "private_shared_history": {
+            "invite_only": True,
+            "history_public_to_subscribers": True,
+            "is_web_public": False,
+            "policy_name": gettext_lazy("Private, shared history"),
```

Comment:
@alya do we want these descriptions to be "Private with shared history" as more of a sentence, rather than this comma format?

Reply:
@timabbott, @alya For context, these strings are taken from what we display to the user in the stream settings UI.

Reply:
Can I see a screenshot? I don't have enough context for where/how this is showing up.

Reply:
![stream_permissions](https://user-images.githubusercontent.com/7251823/145902737-5486dcbd-62cb-4431-b32e-a09353dbe73e.png)

@alya :)

Reply:
I think that having the notification message use the same terms as the stream settings UI would be good, so if we end up changing this, we should probably change it in both places.